navit/navit.c

```diff
@@ -3504,6 +3504,27 @@ void navit_layout_switch(struct navit *n) {
                 && valid_attr.u.num==attr_position_valid_invalid) {
             return; //No valid fix yet
         }
+
```

Comment:
Suggest adding a attr to disable this, as not everybody may like that switch.

Reply:
Added new attributes for navit tag: 

- tunnel_nightlayout              :enable nightlayout in tunnel -  default off
- layout_daynightauto           :enable auto_switch layout - default on
- sunrise_degrees                 :change the degrees after sunrise for calculating the auto switch trigger - default -5